src/content/uk/2020/page-weight.md

```diff
@@ -0,0 +1,156 @@
+discuss: 2054
+results: https://docs.google.com/spreadsheets/d/1wG4u0LV5PT9aN-XB1hixSFtI8KIDARTOCX0sp7ZT3h0/
+featured_quote: Шлях Інтернету від простої, майже педагогічної платформи до інноваційних, складних і надзвичайно інтерактивних застосунків, які стали нормою, рудиментарний показник розміру сторінки приховав за собою більшу історію:&colon; рататуй з ресурсів, кожен з яких впливає на сучасні показники, в свою чергу впливаючи на користувацький досвід.
+featured_stat_1: 1 915
+featured_stat_label_1: Медіанна кількість байтів мобільної сторінки
+featured_stat_2: 916
+featured_stat_label_2: Медіанна кількість байтів зображень мобільної сторінки
+featured_stat_3: 70
+featured_stat_label_3: Медіанна кількість запитів мобільної сторінки
+---
+
+## Вступ
+
+Розмір сторінки — одна з найпростіших доступних метрик. Подібно до того, як нам потрібно стати на ваги для того, щоб оцінити власну вагу (насправді, масу, але ви зрозуміли), завантаження сторінки дозволить оцінити кількість і розмір зібраних та запитаних ресурсів. Але так само, як зміцніли та виросли веб і веб-сторінки, так само зросли й пов’язані з цим метрики, такі як розмір сторінки. Він може вплинути на продуктивність сторінки так само, як ваша власна вага (маса). Цей розділ заглибиться і розшарує веб-сторінки, а також покаже, з чого складається розмір сторінки, який може потенційно нашкодити кінцевому споживачу: вам, мені, нам.
+
+<!-- markdownlint-disable MD018 -->
+#РозмірСторінкиВсеЩеВажливий
+
+#РозмірСторінкиВсеЩеВажливий майже натякає на те, що розмір сторінки не важливий, або ніколи не був. Можливо, він був не важливим, коли запустився базований на тексті [Craigslist](https://uk.wikipedia.org/wiki/Craigslist). Але 25 років тому, коли він був започаткований, в тому самому році був запущений Mosaic 1.0, а пісня Waterfalls від TLC вважалася супер-хітом. Веб зміцнів і так само ресурси. Лише кілька років тому у Twitter обговорювали, як середній розмір веб-сторінки зараз досяг розміру [оригінальної Doom](https://www.wired.com/2016/04/average-webpage-now-size-original-doom/). Багато хто з нас замислився над тим, якими з часом стануть розміри сторінок, включаючи [нашу добру знайому Tammy Everts](https://www.speedcurve.com/blog/web-performance-page-bloat/), але реальність приголомшує. Показники знаходяться на ~4 МБ та 3.7 МБ для настільних/мобільних сторінок відповідно, 75% і вище та шокуючих 7.4 МБ і 6.7 МБ у 90% і вище. Є безліч поганого в тому, щоб мати такі важкі сторінки, наприклад, ймовірність погіршення користувацького досвіду через ненадійні мережі. Сьогодні, не зважаючи на [вивчені десятиліття тому уроки](https://blog.chriszacharias.com/page-weight-matters), ми переживаємо варіації тих самих викликів: попри те, що мережі покращуються, ми працюємо зі значно більшими ресурсами.
```

Comment:
We actually started to external links (except Twitter, Wikipedia and MDN as they are often available in translations). Guessing you started this translation before then as the later ones seem to be done!

```suggestion
#РозмірСторінкиВсеЩеВажливий майже натякає на те, що розмір сторінки не важливий, або ніколи не був. Можливо, він був не важливим, коли запустився базований на тексті [Craigslist](https://uk.wikipedia.org/wiki/Craigslist). Але 25 років тому, коли він був започаткований, в тому самому році був запущений Mosaic 1.0, а пісня Waterfalls від TLC вважалася супер-хітом. Веб зміцнів і так само ресурси. Лише кілька років тому у Twitter обговорювали, як середній розмір веб-сторінки зараз досяг розміру <a hreflang="en" href="https://www.wired.com/2016/04/average-webpage-now-size-original-doom/">оригінальної Doom</a>. Багато хто з нас замислився над тим, якими з часом стануть розміри сторінок, включаючи <a hreflang="en" href="https://www.speedcurve.com/blog/web-performance-page-bloat/нашу добру знайому Tammy Everts</a>, але реальність приголомшує. Показники знаходяться на ~4 МБ та 3.7 МБ для настільних/мобільних сторінок відповідно, 75% і вище та шокуючих 7.4 МБ і 6.7 МБ у 90% і вище. Є безліч поганого в тому, щоб мати такі важкі сторінки, наприклад, ймовірність погіршення користувацького досвіду через ненадійні мережі. Сьогодні, не зважаючи на <a hreflang="en" href="https://blog.chriszacharias.com/page-weight-matters">вивчені десятиліття тому уроки</a>, ми переживаємо варіації тих самих викликів: попри те, що мережі покращуються, ми працюємо зі значно більшими ресурсами.
```

Reply:
My bad. Will watch links from now.